op-e2e/e2eutils/setup.go

```diff
@@ -235,10 +235,24 @@ func Setup(t require.TestingT, deployParams *DeployParams, alloc *AllocParams) *
 	require.Equal(t, deployParams.Secrets.Addresses().SequencerP2P, deployParams.DeployConfig.P2PSequencerAddress)
 	require.Equal(t, deployParams.Secrets.Addresses().Proposer, deployParams.DeployConfig.L2OutputOracleProposer)
 
+	// Create a dependency set when interop or jovian fork is scheduled
+	var dependencySet depset.DependencySet
```

Comment:
TBH, not really sure if this is needed or not? Added it only to pass the existing e2e/unit tests.